changelog.rst

```diff
@@ -3,10 +3,17 @@
 Changelog
 ---------
 
+0.10.5
+~~~~~~
+
+2020-xx-xx • `full history <https://github.com/gorakhargosh/watchdog/compare/v0.10.4...master>`__
+
+- [inotify] Add events for IN_WRITE_CLOSE and IN_NOWRITE_CLOSE (`#690 <https://github.com/gorakhargosh/watchdog/pull/690>`)
```

Comment:
In fact, this is going into v0.10.4, no need for a new version :)

Reply:
Also add youself in the line (in 0.10.4) "Thanks to our beloved contributors: ...".